src/locales/en.json

```diff
@@ -304,9 +304,14 @@
     },
     "audio-extensions": {
       "title": "File type",
-      "mp3": "MP3s",
-      "ogg": "OGGs",
-      "flac": "FLACs"
+      "flac": "FLAC",
+      "mid": "MID",
+      "mp3": "MP3",
+      "oga": "OGA",
+      "ogg": "OGG",
+      "opus": "OPUS",
+      "wav": "WAV",
+      "webm": "WEBM"
     },
     "image-extensions": {
       "title": "File type",
```

Comment:
I guess it would be consistent to name the image extension filter "Extension", too, and remove the `s` at the end of the strings, like JPEG~s~.